tests/k6/tests/performancetest_data/01-create-dialog.js

```
@@ -0,0 +1,19 @@
+import {default as createDialogPayload} from "../serviceowner/testdata/01-create-dialog.js"
+
+function cleanUp(payload) {
+    delete payload.visibleFrom;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)